### ISSUE TITLE
MIssing first column

### ISSUE THREAD

**Issue by davitof** [laurent22/joplin#8201]
## Environment

Joplin version: Joplin 2.10.18 
Platform: Windows 
OS specifics: 11 64 bits

## Steps to reproduce

1. load Joplin in double side-by-side editor
2. observe the first column of the left editor

## Describe what you expected to happen
Obviously, the left column is missing, as if it was hidden by something. 

Simply resizing the window makes the first characters visible again.
![image](https://github.com/laurent22/joplin/assets/22581463/6f9becd8-5bd4-4025-aea1-4766b67506da)

**Comment by Daeraxa** [comment]
Do you have any plugins installed?

**Comment by davitof** [comment]
Good catch ! I disabled Folding Code Editor and Text Colorize and the missing column is back. I did not check which was the culprit.

**Comment by Daeraxa** [comment]
The persistent folding one is the common culprit, I think there is a CSS fix on the forum

**Comment by davitof** [comment]
I verified that Folding Code Editor was the culprit.

Sorry, I am not an expert, to say the least. "Persistent folding", did you mean Folding Code Editor? Or does Folding Code Editor somehow use persistent folding?

I searched the forum and found [this](https://discourse.joplinapp.org/t/persistent-text-folding-in-editor/16183/63). Is this what you were thinking of? I tried anyway. I created a userchrome.css file in the .config\joplin-desktop folder, added the lines from the forum there and restarted Joplin. 

This does indeed solve my issue. Thanks for your help!
